### DISCUSSION TITLE
Welcome to vmware_exporter Discussions!

### DISCUSSION THREAD

**Discussion by pryorda** [pryorda/vmware_exporter#392] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cherryroy** [comment]
Hello Pryorda and all,
thanks for the exporter, it works well with 1 vcenter. when adding additional vcenters my question is I see we can add "?section=limited" to the URL. But how do we edit config.env? I went over all the discussions but couldn't find the answer. can some be more clear on the steps.

**Reply by seowjin1060** [reply]
duplicated issue with https://github.com/pryorda/vmware_exporter/issues/393

just add other target at config file and access to 

http://YOUR_IP:9275/metrics?section=vcenter 2 

or you can  run another process with seperated config files
- run Docker for each vcenter  with other env 
- run multiple background process with -c options
